config/example_gacha.json

```diff
@@ -0,0 +1,6 @@
+{
+	"binatog" : {
+		"play as if its your last blackpink" : 50,
+		"play binatog" : 50
+	}
```

Comment:
Suggestion: Add proper parameter labels/names at it is being interpreted as json.

Reply:
no need imo. this is the same format as aliases.json where the key is the command